ai21/clients/common/chat_base.py

```diff
@@ -89,17 +89,27 @@ def _create_body(
             {
                 "model": model,
                 "system": system,
-                "messages": [message.to_dict() for message in messages],
+                "messages": [message.dict() for message in messages],
```

Comment:
Pydantic should serialize the object automatically when being sent. So we might not need this dict conversion